readthedocs/api/v3/views.py

```diff
@@ -238,7 +252,7 @@ class SubprojectRelationshipViewSet(
     model = ProjectRelationship
     lookup_field = "alias"
     lookup_url_kwarg = "alias_slug"
-    queryset = ProjectRelationship.objects.all()
+    permission_classes = [ReadOnlyPermission | (IsAuthenticated & IsProjectAdmin)]
```

Comment:
APIv3Settings no longer has a default permission class, each view needs to explicitly declare the permissions. This is since each view requires different permissions.